index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500;600;700;800;900&family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+  <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <div class="header-image navbar-item"><img src="images/image_geometry_1.svg" alt="A geometry image"/></div>
```

Comment:
- [ ] The logo should be placed inside an  `a` tag

![a](https://user-images.githubusercontent.com/61562626/133599905-ab8a7907-1da5-48f1-9aae-405373296ef5.jpeg)